api/internal/monkeypatch.js

```diff
@@ -169,7 +170,13 @@ export function init () {
     FileSystemHandle,
     FileSystemFileHandle,
     FileSystemDirectoryHandle,
-    FileSystemWritableFileStream
+    FileSystemWritableFileStream,
+
+    // buffer
+    Buffer,
+
+    // platform detection
+    isSocketRuntime: true
```

Comment:
we can also make it non-configurable, but it's not very important I guess

Reply:
The `install()` function already does that